### PR TITLE
add config for eks publish

### PR DESCRIPTION
**Pull request by dbelev**

### CODE REVIEW

**Comment by bboykov** [comment]
Add the deployment to the provisioner deploy script: https://github.com/Financial-Times/upp-eks-provisioner/pull/19

**Comment by dbelev** [comment]
smartlogic.varnish-api-key lives in global-secrets and I have added it in the global-secrets repo.

**Comment by dbelev** [comment]
Now added in the deploy script

File: helm/k8s-pub-auth-varnish/templates/deployment.yaml [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
     matchLabels:
       app: {{ .Values.service.name }}
   template:
     metadata:
       labels:
         app: {{ .Values.service.name }}
-        visualize: "true" 
+        visualize: "true"
     spec:
+{{- if not .Values.eksCluster }}
```
Comment (bboykov) [inline helm/k8s-pub-auth-varnish/templates/deployment.yaml:25]: I think we agreed that the podAntiAffinity would stay.

Comment (dbelev) [reply helm/k8s-pub-auth-varnish/templates/deployment.yaml]: Antiaffinity is back

**Pull request merged**